src/lib/cimb_translator/LinearDecodePositions.h

```diff
@@ -26,9 +26,9 @@ class LinearDecodePositions
 	CellDrift _drift;
 };
 
-inline LinearDecodePositions::LinearDecodePositions(int spacing, int dimensions, int offset, int marker_size)
-    : _positions(CellPositions::compute(spacing, dimensions, offset, marker_size, 0))
-    , _drift()
+inline LinearDecodePositions::LinearDecodePositions(cimbar::vec_xy spacing, cimbar::vec_xy dimensions, int offset, cimbar::vec_xy marker_size)
+	: _positions(CellPositions::compute(spacing, dimensions, offset, marker_size, 0))
+	, _drift()
```

Comment:
This whitespace change symbolizes my imminent breaking point of finally introducing clang-format into the code base